### ISSUE TITLE
Docker doesn't start

### ISSUE THREAD

**Issue by TessyPowder** [jlesage/docker-nginx-proxy-manager#85]
Half an hour ago i started seeing errors in some of the services this proxy manager manages and noticed the problem with all of them was websockets not working.
So i recreated the container, with the same config directory mapped, but now it doesn't start anymore.
So i mapped the config directory to a empty folder and got this log (its pretty much the same as after recreating, but this time it has to create the database and directory structure, wich it doesn't):
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-app-niceness.sh: executing...
[cont-init.d] 00-app-niceness.sh: exited 0.
[cont-init.d] 00-app-script.sh: executing...
[cont-init.d] 00-app-script.sh: exited 0.
[cont-init.d] 00-app-user-map.sh: executing...
[cont-init.d] 00-app-user-map.sh: exited 0.
[cont-init.d] 00-clean-logmonitor-states.sh: executing...
[cont-init.d] 00-clean-logmonitor-states.sh: exited 0.
[cont-init.d] 00-clean-tmp-dir.sh: executing...
[cont-init.d] 00-clean-tmp-dir.sh: exited 0.
[cont-init.d] 00-set-app-deps.sh: executing...
[cont-init.d] 00-set-app-deps.sh: exited 0.
[cont-init.d] 00-set-home.sh: executing...
[cont-init.d] 00-set-home.sh: exited 0.
[cont-init.d] 00-take-config-ownership.sh: executing...
[cont-init.d] 00-take-config-ownership.sh: exited 0.
[cont-init.d] 00-xdg-runtime-dir.sh: executing...
[cont-init.d] 00-xdg-runtime-dir.sh: exited 0.
[cont-init.d] nginx-proxy-manager.sh: executing...
foreground: warning: unable to spawn /var/run/s6/etc/cont-init.d/nginx-proxy-manager.sh: Exec format error

foreground: fatal: unable to wait for /var/run/s6/etc/cont-init.d/nginx-proxy-manager.sh: No child process
[services.d] stopping services
[services.d] stopping s6-fdholderd...
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] syncing disks.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

I don't know why this is happening. I had a out of Memory problem today, but i guess if that did anything it wouldn't be a problem after recreating the docker. The config seems fine.

# System
- Unraid
- AMD Ryzen 5 2700
- Asrock B450
- newest version of this image on dockerhub

**Comment by TessyPowder** [comment]
I rebooted the system, to see if the OOM had cause any other issues leading to this error, but because Unraid is a ready only OS and the issue persists it has to be some error in this image or the nginx proxy manager itself

**Comment by TessyPowder** [comment]
Because i really need this to work i migrated to the image of jc21.
It's not as easy to set up as this one, but it works.

Has any one else this Problem ? If not we can probably close this.

**Comment by jlesage** [comment]
This is a strange issue...  Did you check the memory usage on your system ?

**Comment by TessyPowder** [comment]
Yes, its under 70%

**Comment by jlesage** [comment]
And is there any memory limit configured for the container ?

**Comment by TessyPowder** [comment]
> And is there any memory limit configured for the container ?

No

**Comment by stopspazzing** [comment]
Curious, have you ran any ram and storage checks to verify nothing is corrected and memory isn't faulty?

**Comment by JonathanTreffler** [comment]
Yes i have. The memory passed the test and the storage had no issues so far

**Comment by stopspazzing** [comment]
> Yes i have. The memory passed the test and the storage had no issues so far

Fully updated bios? No overclocking and proper settings like xmp for proper memory clock speeds etc? Did you clear old cache for docker image before redownload ing it?

**Comment by JonathanTreffler** [comment]
> Fully updated bios? No overclocking and proper settings like xmp for proper memory clock speeds etc? Did you clear old cache for docker image before redownload ing it?

Bios and XMP are fine.
I didn't clear the docker image cache, but i don't think the  image was corrupted, because that would probably be noticed by some hash checks.

**Comment by jlesage** [comment]
Re-creating the docker image may worth the try, I already saw some weird issues being fixed by this.

Also, do you see the same error if you run `docker run --rm jlesage/nginx-proxy-manager` ?
